WordPress/src/main/java/org/wordpress/android/ui/notifications/NotificationsDetailListFragment.java

```diff
@@ -173,11 +173,15 @@ public void showSitePreview(long siteId, String siteUrl) {
 
     private final UserNoteBlock.OnGravatarClickedListener mOnGravatarClickedListener = new UserNoteBlock.OnGravatarClickedListener() {
         @Override
-        public void onGravatarClicked(long siteId, long userId) {
+        public void onGravatarClicked(long siteId, long userId, String siteUrl) {
             if (!isAdded()) return;
 
             NotificationsActivity notificationsActivity = (NotificationsActivity)getActivity();
-            notificationsActivity.showBlogPreviewActivity(siteId, null);
+            if (siteId == 0 && !TextUtils.isEmpty(siteUrl)) {
+                notificationsActivity.showWebViewActivityForUrl(siteUrl);
+            } else {
+                notificationsActivity.showBlogPreviewActivity(siteId, siteUrl);
```

Comment:
Yeah, I thought that would be good, so they see the error toast message from the blog preview instead of nothing happening.